src/native/libs/build-local.sh

```diff
@@ -13,6 +13,27 @@
 SHIM_SOURCE_DIR="$1"/native/src
 INTERMEDIATE_OUTPUT_PATH="$2"
 TARGET_LIBRARY="$3"
+EMBED_ICU_DATA_PATH="$4"
+
+if [ -n "$EMBED_ICU_DATA_PATH" ]; then
+    header="${INTERMEDIATE_OUTPUT_PATH}embedded_icu_data.h"
+
+    cat <<EOF > "$header"
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
```suggestion
```
Auto-generated files do not need to have license. (Most auto-generated files in the build do not have license.) 

In this particular case, this file is redistributing the data from EMBED_ICU_DATA_PATH in a different format, so .NET Foundation license is not quite right. It would need to be the license that the data file came with.